### PR TITLE
Failing tests

### PR DESCRIPTION
**Pull request by Janpot**
Added some failing tests for scss

### CODE REVIEW

**Comment by varya** [comment]
These tests are made for SCSS whereas Bootstrap issues are in SASS (this is different). The SASS things are covered in #45 and #46.

**Comment by Janpot** [comment]
If I'm not mistaken, the official bootstrap-sass project is built in scss.

**Comment by varya** [comment]
@Janpot Looks like they supported 2 syntaxes. At least they mention both in the documentation. In this case my PRs are not enough and I need to do the same for SCSS as well.

File: test/scss/function/2.p [hunk 1]
```diff
@@ -0,0 +1,5 @@
+['function', 
+  ['ident', 'fn'],
```
Comment (tonyganch) [inline test/scss/function/2.p:2]: In the `2.scss` file below ident is actually `f` and not `fn`

Comment (Janpot) [inline test/scss/function/2.p:2]: @tonyganch oops that's right, it's not what makes the test fail though.

**Comment by tonyganch** [comment]
Fixed the test and merged into `dev`

**Pull request closed without merging**